docs/modules/deep-equal.md

```diff
@@ -0,0 +1,13 @@
+# deep-equal
+
+`deep-equal` has [roughly 50 dependencies](https://npmgraph.js.org/?q=deep-equal) while alternatives have zero.
```

Comment:
i would reword this to be less specific about the number of dependencies i think (as it may change)

maybe just something like `deep-equal has various lighter weight and faster alternatives`

Reply:
thanks. good point. I've updated it

I also confirmed that saying faster is correct: https://www.npmjs.com/package/fast-deep-equal#performance-benchmark